update to react 17

npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.6.0" from redux-injectors@1.3.0
npm ERR! node_modules/redux-injectors
npm ERR!   redux-injectors@"*" from the root project

had to do this.. also react-boilerplate-cra still on version 16
```npm i redux-injectors --force```

--- a/package.json
+++ b/package.json
@@ -57,8 +57,8 @@
     "regenerator-runtime": "0.13.5"
   },
   "peerDependencies": {
-    "react": "^16.6.0",
-    "react-dom": "^16.6.0",
+    "react": "^16.6.0 || ^17.0.0",
+    "react-dom": "^16.6.0 || ^17.0.0",
     "react-redux": "^7.1.0"
   },
   "dependencies": {

